### ISSUE TITLE
Extremely unhelpful cadence error reports

### ISSUE THREAD

**Issue by rahul-kothari** [onflow/cadence#301]
Hi I am using JS-SDK to send cadence transactions and scripts.

I have various transactions and scripts that worked perfectly fine on playrgound 1-2 months back but now are showing some of the weirdest errors.

Here are some of these weird errors that make no sense:
### Error 1
```
Failed to execute the script on the execution node: rpc error: code = Internal desc = failed to execute script: failed to execute script at block (134d3a0cd56476a21b0409542a1faf4d3a51dcd0a21e819918b8e398d067fd39): Execution failed:\nParsing failed:\nexpected token '{'\n:\n    expected token '{'\n"
```
The actual error was that I had used `UFix64[]` instead of [UFix64]`. So it is a bit misleading.

### Error 2:
```
address is empty
```
* Doesn't mention which address. The code refers to 4 addresses (2 contract imports and 2 user accounts) => simple transfer token example that was picked up and slightly modified from the examples.
* And actually the addresses were in fact there (confirmed from a print statement).

### Error 3:
```
Failed to execute the script on the execution node: rpc error: code = Internal desc = failed to execute script: failed to execute script at block (371ce93c2826d947002fe46872e5fd4ddd778c12732cc809914b2bbd49f34f36): Execution failed:\nParsing failed:\nunrecognized character: U+000D\n:\n    unrecognized character: U+000D\n"
```
* This happens in my playground too (not just in my vscode). Removing CRLF from playground code is not straight forward. 
* Would be helpful if it could mention which line of code it encountered the character.

### Error 4:
The error stack trace for all of these was the same - it doesn't mention the line numbers or anything useful in my code. In fact it doesn't even show which file in my code it touched:
```
[ERROR] at Object.end
my-flow-api\\node_modules\\@onflow\\send\\dist\\send.js:1:379)\n    
at my-flow-api\\node_modules\\@improbable-eng\\grpc-web\\dist\\grpc-web-client.js:1:24910\n    
at Array.<anonymous> (my-flow-api\\node_modules\\@improbable-eng\\grpc-web\\dist\\grpc-web-client.js:1:5649)\n   
at e (my-flow-api\\node_modules\\@improbable-eng\\grpc-web\\dist\\grpc-web-client.js:1:1531)\n    
at Timeout._onTimeout (my-flow-api\\node_modules\\@improbable-eng\\grpc-web\\dist\\grpc-web-client.js:1:1645)\n    
at listOnTimeout (internal/timers.js:549:17)\n    at processTimers (internal/timers.js:492:7)
```

**Comment by turbolent** [comment]
1. I added this to https://github.com/onflow/cadence/issues/294
2. Can you please provide more context where this occurs and the code that leads to that error?
   You mentioned on Discord you are using the JS SDK/FCL – can you confirm this results in the same error when not using FCL, just with e.g. the VS Code extension and the emulator?
3. This is fixed by #304 
4. This is a missing feature in the Access API and the JS SDK/FCL, can you please report it there?

**Comment by rahul-kothari** [comment]
2 was fixed with the new fcl version.
3 has yet to be added on playground.
